files/usr/share/cinnamon/cinnamon-settings/bin/Spices.py

```diff
@@ -165,8 +166,8 @@ def __init__(self, collection_type, window=None):
         self.settings.connect('changed::%s' % self.enabled_key, self._update_status)
 
         if self.themes:
-            self.install_folder = '%s/.themes/' % (home)
-            self.spices_directories = (self.install_folder, '%s/.themes/' % (home))
+            self.install_folder = os.path.join(xdg_data_home, 'themes')
+            self.spices_directories = (self.install_folder, self.install_folder)
```

Comment:
Why is spices_directories a tuple of the same thing twice?

Reply:
Answer: it shouldn't be, now fixed via #8828 e2690547e0baeeedfd3ce4add9645d86d1cb66a0

Hmm...